pkg/testing/tools/fleet_package_policy_api.go

```diff
@@ -0,0 +1,106 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
```

Comment:
Should the contents of this file be moved into https://github.com/elastic/elastic-agent-libs/blob/main/kibana/fleet.go where the other Fleet API client code lives? Or we could go the opposite direction too.  But either way, I think we should keep all the Fleet API client code together.

Reply:
Ah, just noticed that you have a `TODO` in this file to move this code into `elastic-agent-libs`.  Any objections to doing that first, before using it from the test in this PR here?

Reply:
My preferences is to do it as a follow up, because it has already taken me far too long to get this PR into shape. I don't want to risk delaying it longer considering this test is finding bugs and we the tamper protection work is doing a lot of big changes that need it.